packages/reaction-collections/package.js

```diff
@@ -24,6 +24,7 @@ Package.onUse(function (api) {
   api.use("ongoworks:security@1.3.0");
   api.use("ongoworks:bunyan-logger@2.5.0");
   api.use("alanning:roles@1.2.13");
+  api.use("tmeasday:publish-counts@0.7.3");
```

Comment:
This package doesn't seem to be very well maintained,(source has deprecated Meteor code) and warns ```This package is designed primarily for correctness, not performance. That's why it's aimed at counting smaller datasets and keeping the count instantly up to date.```  Is this the best choice?